pkg/kubelet/cm/topologymanager/topology_manager.go

```diff
@@ -37,6 +37,17 @@ const (
 	// present on a machine and the TopologyManager is enabled, an error will
 	// be returned and the TopologyManager will not be loaded.
 	maxAllowableNUMANodes = 8
+	// containerTopologyScope specifies the TopologyManagerScope per container.
+	containerTopologyScope = "container"
+	// podTopologyScope specifies the TopologyManagerScope per pod.
+	podTopologyScope = "pod"
```

Comment:
Where are the policies defined? I would expect these to go next to that.

Reply:
You mean that it should be placed in different file? Each policy name is defined in separate file. It makes sense to me to leave it here.

Reply:
`topologyScope` is fixed when kubelet starts using `--topologyManagerScope=[pod|container]` option.